src/SkillsForm.jsx

```diff
@@ -16,23 +15,18 @@ class SkillForm extends React.Component {
     super(props);
     this.state = {
       skill_name: "",
-      skill_type: ""
+      skill_type: skillTypes[0].value
```

Comment:
The problem is that the selector shows the first item automatically, and not setting it here means that it can be unset when the user sees it as being set.